plugins/org.jboss.tools.jst.angularjs/about.properties

```diff
@@ -0,0 +1,2 @@
+blurb=JBoss Tools Java Standard Tools AngularJS\n\nVersion\: {featureVersion}\n\n(c) Copyright (c) Red Hat, Inc., contributors and others 2013.  All rights reserved.\nVisit http\://jboss.org/tools
```

Comment:
Don't understand what Java is doing here. It would rather have:
JBoss Tools JavaScript Tools for AngularJS
to be in line with org.jboss.tools.jst.js